docs/application/dotnet/guides/text-input/input-method.md

```diff
@@ -88,7 +88,7 @@ To start the IME application life-cycle:
 
 ## Use Floating Keyboard
```

Comment:
Please can you change "Use" to "Using" to maintain consistency with the entire document.